setup.py

```diff
@@ -32,8 +32,8 @@
     author = "Trackpy Contributors",
     author_email = "daniel.b.allan@gmail.com",
     url = "https://github.com/soft-matter/trackpy",
-    install_requires = ['numpy>=1.8', 'scipy>=0.12', 'six>=1.8',
-	                'pandas>=0.13', 'pyyaml', 'matplotlib'],
+    install_requires = ['numpy>=1.9', 'scipy', 'six>=1.8',
+	                'pandas', 'pyyaml', 'matplotlib'],
```

Comment:
Did you remove the version requirements for scipy and pandas because `numpy>=1.9` is sufficient to pin those other packages? If not, I think they are useful, since some scientific users have really weird environments and installation methods.

Reply:
I put back the version requirements to match the legacy env on travis, just to be safe.